src/components/Table/TableBody/TableBody.types.ts

```diff
@@ -13,4 +13,12 @@ export interface TableBodyProps {
   className?: string;
   /** The system prop that allows defining system overrides as well as additional CSS styles. */
   sx?: SxProps<Theme>;
+  /** If `true`, the table row dividers is shown.
+   * @default true
+   */
+  rowDividers?: boolean;
+  /** If `true`, the table col dividers is shown.
```

Comment:
Извиняюсь, немного ошибся в прошлый раз, тут же множетсвенное число =)
```suggestion
  /** If `true`, the table col dividers are shown.
```